### DISCUSSION TITLE
How can I add custom resolutions and/or change scaling?

### DISCUSSION THREAD

**Discussion by BuyagaDodzyo** [mir-diablo-ii-tools/SlashGaming-Diablo-II-Free-Resolution#137] [Q&A]
Have prepackaged mod files in which has "SGD2Res.dll", "SGD2Res.mpq" and "SGD2Resolution.json" files included. I figured out that in "SGD2Resolution.json" we have: 
`        "Ingame Resolutions": [
            "640x480",
            "800x600",
            "1280x720"`
Which meaned that game scale to "1280x720" after character loaded. If my goal are to have something between "1280x720" and "800x600" I need just add there propper numbers? Am I correct?
I don't find about it enywhere and f*ck with those files long enough to ask before I broke something and be forced to check all files to make this mod launchable again. Please help.

**Comment by systemdissue** [comment]
Yes, you need to add your own resolution(s) between 1280x720 and 800x600. Don't forget to use "," after every value except the last.

Example:

"Ingame Resolutions": [ "640x480", "800x600", "1068x600", "1280x720"

Also... did you find how to change Main Menu Resolution? It has "800x600" by default but when I try to change it and launch the game - I have just black screen instead of a main menu (but listen main menu's soundtrack).
